src/llm_vm/onsite_llm.py

```diff
@@ -303,6 +304,36 @@ def model_loader(self):
         return LlamaForCausalLM.from_pretrained(self.model_uri)
     def tokenizer_loader(self):
         return LlamaTokenizer.from_pretrained(self.model_uri)
+    
+@RegisterModelClass("quantized-llama")
+class Quantized_Llama(Base_Onsite_LLM):
+    """
+    Class for running quantized Llama instances that use GGML
+
+    Attributes:
+        model_uri (str): Hugging Face Endpoint for LLM
```

Comment:
are we gonna need model_file_name for this?
how does accessing a local model work vs remote model?
with the others, the URI could be a local file path i think, though its probably not very robust

Reply:
It looks like ctransformers behaves the same way: https://github.com/marella/ctransformers#classmethod-automodelforcausallmfrom_pretrained